tests/functional/test_accounts.py

```diff
@@ -291,7 +293,9 @@ def test_impersonate_not_implemented(accounts):
         _ = accounts[test_address]
 
 
-def test_contract_as_sender_non_fork_network(contract_instance):
+def test_contract_as_sender_non_fork_network(mocker, contract_instance):
+    mocked_provider = mocker.patch("ape_test.provider.LocalProvider")
+    mocked_provider.unlock_account.side_effect = NotImplementedError
```

Comment:
I think by default, `unlock_account` is not implemented in the ethtester provider, no?

Reply:
nevermind, I see you implemented it!